tests/test_workflow.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+from pytest_workflow.workflow import Workflow
+
+
+def test_stdout():
+    workflow = Workflow(Path("echo"), ["moo"])
+    workflow.run()
+    assert workflow.stdout == b"moo\n"
```

Comment:
:cow:

Reply:
Unfortunately, I can be pretty sure `echo` will be on every test machine, but this does not go for `cowsay` :disappointed: .